src/main/java/codeu/model/store/basic/IndexStore.java

```diff
@@ -0,0 +1,91 @@
+package codeu.model.store.basic;
+
+import codeu.model.data.Message;
+import codeu.model.data.Conversation;
+import codeu.model.data.User;
+import java.util.HashSet;
+import java.util.Hashtable;
+import java.util.Set;
+
+public class IndexStore {
```

Comment:
Could you add a quick description like "Stores mapping of words to messages that contain the word" or "Stores mappings from key words to search results to make search function quick" Or whatever you think makes sense